src/components/Restaurants/RestaurantReview.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa'; 
+import './RestaurantList.css';
+
+const SearchBar = () => (
+  <header className="header">
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="Restaurants"
+        className="searchInput"
+        aria-label="Search for restaurants"
+      />
+      <input
+        type="text"
+        placeholder="Address, neighborhood"
+        className="locationInput"
+        aria-label="Location"
+      />
+      <button className="searchButton">
+        <FaSearch />
+      </button>
+    </div>
+  </header>
+);
+
+const Filters = () => (
+  <div className="filters">
+    <div className="filterSection">
+      <button className="button">$</button>
+      <button className="button">$$</button>
+      <button className="button">$$$</button>
+      <button className="button">$$$$</button>
+    </div>
+    <div className="filterSection">
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Open Now
+      </label>
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Reservations
+      </label>
+    </div>
+    <div className="filterSection">
+      <button className="categoryButton">Mexican</button>
+      <button className="categoryButton">Sandwiches</button>
+    </div>
+  </div>
+);
+
+const RestaurantEntry = ({ imageSrc, name, tags, rating, reviewCount, url }) => (
+  <div className="restaurantEntry">
+    <img src={imageSrc} alt={name} className="restaurantImage" />
```

Comment:
Notes:
- If we replace the imageSrc with some url, for instance https://s3-media0.fl.yelpcdn.com/bphoto/td7RDAytLoNuPMlCQ6IuMw/348s.jpg, the image looks huge. Need to set the width to 180px
- image should have a margin-right about 15px
- image has small border radius (4px)